### DISCUSSION TITLE
Tiempo distinto en Valgrind

### DISCUSSION THREAD

**Discussion by GuilleStankeS** [IIC2133-PUC/2024-2#45] [Tarea 0] [answered]
Hola, tengo una duda respecto al tiempo de ejecución...
Cuando ejecuto mi código con valgrind, con un input.txt grande, se demora bastante en terminar de ejecutarse el código. Sin embargo, cuando ejecuto "time ./puffle input.txt output.txt" (no con valgrind), el tiempo está bien (se demora poco)... De la misma forma, ejecutando solamente "./puffle input.txt output.txt" también se demora poco... Esto significa que está correcto? o el tiempo de ejecución con valgrind también debe ser menor a 5 segundos???

**Comment by tomastrivino** [comment, marked as answer]
Hola,
Véase #34
Éxito!
